widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterController.kt

```diff
@@ -59,17 +62,26 @@ internal class MessageCenterController(
     private var state = MessageCenterState()
 
     override fun setConfiguration(uiTheme: UiTheme?, configuration: GliaSdkConfiguration?) {
+        setQueueIds(configuration?.queueIds)
+
         serviceChatHeadController.setBuildTimeTheme(uiTheme)
         serviceChatHeadController.setSdkConfiguration(configuration)
     }
 
+    private fun setQueueIds(queueIds: List<String>?) {
+        engagementConfigUseCase(ChatType.SECURE_MESSAGING, queueIds ?: emptyList())
+    }
+
     override fun setView(view: MessageCenterContract.View) {
         this.view = view
+    }
 
+    override fun initialize() {
         if (isAuthenticatedUseCase()) {
             initComponents()
         } else {
             dialogController.showUnauthenticatedDialog()
+            Logger.w(TAG, "Secure Messaging is unavailable because the visitor is not authenticated.")
```

Comment:
It seems to me that 'info' level is enough for this event. Visitor just has to authenticate and it's a normal case/flow. It's not something that we should definitely keep an eye on.

Reply:
Maybe you're right.
@EgorovEI, what do you think? Let's change it to `info`.